### DISCUSSION TITLE
Intersect broke my navigation

### DISCUSSION THREAD

**Discussion by laraib07** [alpinejs/alpine#4020] [1. Help] [answered]
I am working on my [portfolio](https://laraib07.github.io/portfolio/). I am using intersect to add entry animation to my sections. Also I am using top navbar to navigate to different section of the page. But the issue is that intersect broke the navigation. When I tap on nav links to navigate to the mentioned section, it stops before reaching the section.

**Comment by ekwoka** [comment]
Because the spot on the page that id is at changes when the things animate in, probably.

Also, you shouldn't really have multiple H1 tags on a page.

**Reply by laraib07** [reply]
Is there any workaround for it?

Thanks for the H1 tag tip :)

**Reply by ekwoka** [reply, marked as answer]
Not have the things be hidden.

Just invisible.

**Reply by laraib07** [reply]
Thanks @ekwoka 
I also came to the same conclusion.
Earlier I was using `x-show` directive. This was causing all the issue. Now I am using CSS classes and `x-bind` to change opacity.
